### PR TITLE
Update `helios hosts` docs with new operators

### PR DESCRIPTION
**Pull request by davidxia**

### CODE REVIEW

**Comment by davidxia** [comment]
@mattnworb

**Comment by davidxia** [comment]
@spotify/helios-team

File: docs/user_manual.md [hunk 1]
```diff
@@ -71,8 +71,12 @@ Register and unregister with `helios [un]register <hostname> <unqiue ID>`.
 ### List Agents
 
 List agents with `helios hosts [optional hostname pattern]`. If your agents have labels like
-`key=value` (see below on how to label agents), you can filter on labels with 
-`helios hosts -l key1=value1 -b key2=value2`.
+`key=value` (see below on how to label agents), you can use host selector expressions like
+`helios hosts -l key1=value1 -l key2!=value2 -l key3 in (value3a, value3b)`.
```
Comment (mattnworb) [inline docs/user_manual.md]: #957 changed the CLI argument to be `-s` or `--selector` rather than `-l`

File: docs/user_manual.md [hunk 2]
```diff
@@ -71,8 +71,12 @@ Register and unregister with `helios [un]register <hostname> <unqiue ID>`.
 ### List Agents
 
 List agents with `helios hosts [optional hostname pattern]`. If your agents have labels like
-`key=value` (see below on how to label agents), you can filter on labels with 
-`helios hosts -l key1=value1 -b key2=value2`.
+`key=value` (see below on how to label agents), you can use host selector expressions like
+`helios hosts -s key1=value1 -s key2!=value2 -s key3 in (value3a, value3b)`.
```
Comment (mattnworb) [inline docs/user_manual.md]: nitpick but you have to quote the parenthesis

`helios hosts -s key1=value1 -s key2!=value2 -s key3 in (value3a, value3b)`

**Comment by mattnworb** [comment]
👍

**Comment by gimaker** [comment]
:+1:

File: docs/user_manual.md (hunk 2 above)
Comment (davidxia) [inline docs/user_manual.md]: Ay, true.

**Pull request merged**